qiskit/compiler/transpile.py

```diff
@@ -108,6 +110,8 @@ def transpile(circuits: Union[QuantumCircuit, List[QuantumCircuit]],
 
                     [qr[0], None, None, qr[1], None, qr[2]]
 
+        layout_method: Name of layout selection pass ('trivial', 'dense', 'noise_adaptive')
+        routing_method: Name of routing pass ('basic', 'lookahead', 'stochastic')
```

Comment:
Can we a add `routing_method='none'` here? i.e. "I want to manually map my circuit and I don't want the routing pass to run, even if I get it wrong". If it's too much, I can open a separate issue for it.

Reply:
you could achieve this by doing e.g.
```
transpile(circuit, backend, coupling_map=None)
```
to override the backend coupling map to be fully connected, and no routing will be done.

Reply:
Right now, this doesn't work (since `coupling_map=None` is the default, it's the same as not providing a coupling_map, i.e. we'd just fetch the coupling_map from the backend). I'll open an issue and then we can figure out how to implement it.